File: 프로그래머스/2/140107. 점 찍기/점 찍기.py
from math import sqrt
def solution(k, d):
    
    answer = 0
    lim = d**2
    for x in range(0, d+1, k):
        y = sqrt(lim - x**2)
        cnt = (y // k) + 1
        answer += cnt
        
    return answer

File: 프로그래머스/2/140107. 점 찍기/점 찍기.js
function solution(k, d) {
    var answer = 0;
    
    const lim = d ** 2
    
    for (let x = 0; x < d+1; x += k) {
        const y = Math.sqrt(lim - (x ** 2))
        const cnt = (Math.floor(y / k) + 1)
        answer += cnt
    }
    
    
    return answer;
}